Library.Business/Mapping/ModelMappingProfile.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using Library.Business.DTO.Book;
+using Library.Business.DTO.Country;
+using Library.Business.DTO.Person;
+using Library.Business.DTO.PublishingHouse;
+using Library.DAL.Entities;
+using System.Linq;
+
+namespace Library.Business.Mapping
+{
+    public class ModelMappingProfile : Profile
+    {
+        public ModelMappingProfile()
+        {
+            // To review and edit
+            CreateMap<Book, BookDto>().ForMember(destination => destination.AuthorIds, conf => conf.MapFrom(x => x.BookAuthors.Select(y => y.AuthorId).ToList())); // Проверитьб нужен ли ToLLIst
+            CreateMap<BookDto, Book>();
+
+            CreateMap<CreateBookDto, Book>();
+
+            CreateMap<Country, CountryDto>();
+            CreateMap<CountryDto, Country>();
```

Comment:
U can use
.ReverseMap();

Reply:
Also, you can skip creating DTO models in the business layer.

Reply:
It has to be done at API layer?
My thought was - as Service methods are at Business, so it should be on same level